service-commands/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "dependencies": {
     "@audius/libs": "^1.1.16",
+    "@solana/spl-token": "^0.1.4",
```

Comment:
where is this coming from?

Reply:
Sorry, this is from one of the scripts I have on local service commands, will remove